app/views/jobs/_form.html.erb

```diff
@@ -1,15 +1,13 @@
-<%= form_for [@company, @job] do |f| %>
-  <%= f.label :title %>
-  <%= f.text_field :title %>
+<%= link_to 'Add new category', new_category_path, class: "btn btn-default" %>
+<hr />
 
-  <%= f.label :description %>
-  <%= f.text_field :description %>
+<%= bootstrap_form_for [@company, @job] do |f| %>
 
-  <%= f.label :level_of_interest %>
-  <%= f.number_field :level_of_interest %>
-
-  <%= f.label :city %>
-  <%= f.text_field :city %>
+  <%= f.collection_select :category_id, Category.order(:title), :id, :title, hide_label: true %>
```

Comment:
where's an even better place for  `Category.order(:title)`? what part of MVC is responsible for this?